charts/prometheus/templates/server/deploy.yaml

```diff
@@ -110,21 +110,32 @@ spec:
           {{- if .Values.server.baseURL }}
             - --web.external-url={{ .Values.server.baseURL }}
           {{- end }}
+          {{- if .Values.server.listenOnLocalhost }}
```

Comment:
Imho if we make use of "--web.listen-address" it should be configurable and not bound to localhost only.

Reply:
Thanks for the feedback. I got similar feedback from one of my colleagues as well. I did the change now to 
```yaml
listenOnHost: localhost
```
so users can freely configure the host name. The port number 9090 is not configurable.

Reply:
Why not also make the port configurable like?

```
listen:
  host:
  port:
```

Reply:
I like the config style:
```yaml
listen:
  host: ...
```
because it is extendable. However I'd rather not implement switching the port in this PR, because the 9090 is hard-coded in so many places. Tomorrow I will adjust the values.yaml according to your suggestion.